test/e2e/scheduling/predicates.go

```diff
@@ -952,7 +952,7 @@ func createPausePod(ctx context.Context, f *framework.Framework, conf pausePodCo
 }
 
 func runPausePod(ctx context.Context, f *framework.Framework, conf pausePodConfig) *v1.Pod {
-	return runPausePodWithTimeout(ctx, f, conf, framework.PollShortTimeout)
+	return runPausePodWithTimeout(ctx, f, conf, f.Timeouts.PodStart)
```

Comment:
The parameter `timeout` in `runPausePodWithTimeout` seems not necessary any more. Or let's use `framework.PodStartTimeout`, the timeout in framework can be customized.

Reply:
Makes sense, we can do that. Though, my understand is we should use `f.Timeouts...` vs `framework.PodStartTimeout` because only `f.Timeouts` are customizable and `framework.` based timeouts are marked as deprecated.

https://github.com/kubernetes/kubernetes/blob/e150be6872d4972c2c6b886fcd8d185ddd35652c/test/e2e/framework/util.go#L64

Reply:
Updated based on your suggestion, ptal.

Reply:
Ah, didn't notice they're deprecated. 
/lgtm